JuliaGPU/session1/Gauri/q2.jl

```diff
@@ -0,0 +1,14 @@
+#2. Collatz Sequence of a no.
+function CollatzSequence(N)
+    while N!=1
+        println("$N->")
```

Comment:
the function ```print``` doesn't add a newline by default. I think that is the function you meant to use here.